### ISSUE TITLE
SteamVR doesn't work on macOS Catalina 10.15

### ISSUE THREAD

**Issue by n1n3b1t** [ValveSoftware/openvr#1152]
macos_default version shows the grid after launching steamvr, but stuck when trying to launch unity, or any other content.
macos_beta version just crashes itself and steam on each launch.

Is there any roadmap on macOS support? Also Index support would be nice

**Comment by esummers** [comment]
It would be nice to see a roadmap.  I would love to see this working better with MetalKit, Unity, and Unreal on Mac.  Unfortunately if the rumors are true, Steam is putting zero effort in the Mac port at the moment.

**Comment by tomgoddard** [comment]
SteamVR macos_default version is working for me with a Vive and macOS Catalina (10.15 official release), using a VR molecular visualization program ChimeraX.  This is on a 2017 iMac (iMac18,3) with Radeon Pro 580 graphics (no eGPU).  I just updated this machine from Mojave 10.14.6 and on that older macOS the Vive Pro was working fine with SteamVR macos_default and no eGPU.  But in Catalina the Vive Pro is no longer working, SteamVR is giving Headset not connected properly (error 208).  The Vive Pro is working with this same iMac if I hook it up to an eGPU (Sonnet Breakaway Box 550 with Radeon Vega 56 graphics) and use SteamVR macos_default.  I have only tested the ChimeraX VR application.

**Comment by tomgoddard** [comment]
My previous comment about the Vive Pro not working with Catalina when directly connected to the iMac (no eGPU) was apparently caused by the USBC to mini-displayport cable I was using.  I switched that cable and now the Vive Pro works directly to the iMac.  The working cable I believe is cable matters 201076 while the one that is not working is cable creation CD0498.  The non-working cable also has rather poor reviews on Amazon, although I believe it is the cable I have used successfully with the Vive Pro in the past (hard to be sure since the VR headsets get moved around).  Also there is a bit of uncertainty about these cable models since the cables are not marked with unique identifying information.

**Comment by tomgoddard** [comment]
In summary both the Vive and Vive Pro are working for me with macOS Catalina 10.15 (official release) using SteamVR maosx_default, the current version 1539100633, Built Oct 9, 2018 at 08:57, which is one year old (surprising that it still functions) connected directly to an 2017 iMac (model 18,3) with Radeon Pro 580 graphics, and also both work when connected to an eGPU, Sonnet Breakaway Box 550 with Radeon Pro Vega 56 graphics.

**Comment by lukasmaxim** [comment]
> In summary both the Vive and Vive Pro are working for me with macOS Catalina 10.15 (official release) using SteamVR maosx_default, the current version 1539100633, Built Oct 9, 2018 at 08:57, which is one year old (surprising that it still functions) connected directly to an 2017 iMac (model 18,3) with Radeon Pro 580 graphics, and also both work when connected to an eGPU, Sonnet Breakaway Box 550 with Radeon Pro Vega 56 graphics.

Hey tomgoddard! Glad to see I'm not the only one where SteamVR seems to not be working. Since you said that the cable made it work for you, I ordered it and still cannot get Unity to work properly. Are you using the Pro Lighthouses, or the standard ones? For me that is what apparently makes all the difference. Also, did you try out SteamVR in combination with Unity? 

Thanks!

**Comment by tomgoddard** [comment]
Yes, as noted on my Mac SteamVR page the newer basestation 2 model does not work, although older base station 2 and original base station 1 do work.

	http://www.cgl.ucsf.edu/chimera/data/mac-vr-nov2018/mac-vr.html

"September 30, 2019: There are now two versions of Vive basestation 2 and the more recent version does not work with macOS 10.14 (Mojave). I believe the more recent version came out around March 2019. The more recent version has Valve written on the back while the older version has SteamVR as shown in the picture (right). Both are basestation 2.0, model 1004. I obtained the new basestations under warranty replacement from HTC -- when I asked for the older basestation 2.0 style they said it is not made anymore. The original Vive basestation 1.0 models continue to work with Vive and Vive Pro on macOS."

I have not tried Unity on macOS with SteamVR.  All my Unity projects are on Windows only.  I could be that Unity with SteamVR has special problems.  So all of my tests have been with a molecular viewer app ChimeraX that uses Python and pyopenvr to connect to SteamVR.

  Tom
